reframe/frontend/cli.py

```diff
@@ -368,6 +372,24 @@ def main():
     printer.colorize = site_config.get('general/0/colorize')
     printer.inc_verbosity(site_config.get('general/0/verbose'))
 
+    if options.upgrade_config_file is not None:
+        configs = options.upgrade_config_file.split(':', maxsplit=1)
+        old_config = configs[0]
+        new_config = None if len(configs) == 1 else configs[1]
```

Comment:
You can write this in two lines only:

```suggestion
        old_config, *new_config = options.upgrade_config_file.split(':', maxsplit=1)
        new_config = new_config[0] if new_config else None
```